### PR TITLE
feature: use the viewer to render single file shares

### PR DESCRIPTION
**Pull request by max-nextcloud**
Still some issues with positioning and sizing of the image and video.

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]
:elephant:

File: apps/files_sharing/templates/public.php [hunk 1]
```diff
@@ -60,31 +60,21 @@
 			<?php if (isset($_['folder'])): ?>
 				<?php print_unescaped($_['folder']); ?>
 			<?php else: ?>
-				<?php if ($_['previewEnabled'] && substr($_['mimetype'], 0, strpos($_['mimetype'], '/')) == 'audio'): ?>
```
Comment (PVince81) [inline apps/files_sharing/templates/public.php:63]: what about files where there is no preview / viewer possible ?

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
triggered a rebase to bump CI

**Review by CarlSchwan — APPROVED** [review]

**Pull request merged**